bin/generate-gutenboarding-design-thumbnails.js

```diff
@@ -13,29 +13,60 @@
  * GENERATE THUMBNAILS:
  *
  * `node ./bin/generate-gutenboarding-design-thumbnails.js`
+ * or
+ * `node ./bin/generate-gutenboarding-design-thumbnails.js all`
+ * `node ./bin/generate-gutenboarding-design-thumbnails.js mag16`
+ * `node ./bin/generate-gutenboarding-design-thumbnails.js es`
+ * `node ./bin/generate-gutenboarding-design-thumbnails.js fr,de,zh-cn`
```

Comment:
It would be nice to be able to pass a second arg to limit screenshots to a single design, or possibly even just allow specification of a plain url.